.zombienet-tests/0002-parachain.feature

```diff
@@ -0,0 +1,10 @@
+Description: Simple Parachain Smoke Test
+Network: ./0002-parachain.toml
+Creds: config
+
+alice: is up
+bob: is up
+charlie: is up
+dave: is up
+alice: parachain 100 is registered within 225 seconds
+alice: parachain 100 block height is at least 3 within 200 seconds
```

Comment:
This doesn't test anything in substrate-connect?

Reply:
At the moment `zombienet` is not providing the tmp path where the parachain's chainspec exists. The "network info" we have so far is the one below:
```json
{
  "chainSpecPath": "/tmp/zombie-0f364056fe48d2d37b9c79e6f374504b_-70536-6qZpufzfhHyo/wococo-local.json",
  "relay": [
    {
      "name": "alice",
      "wsUri": "ws://127.0.0.1:36755",
      "prometheusUri": "http://127.0.0.1:35527/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    {
      "name": "bob",
      "wsUri": "ws://127.0.0.1:40181",
      "prometheusUri": "http://127.0.0.1:45287/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    {
      "name": "charlie",
      "wsUri": "ws://127.0.0.1:33939",
      "prometheusUri": "http://127.0.0.1:45187/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    {
      "name": "dave",
      "wsUri": "ws://127.0.0.1:33299",
      "prometheusUri": "http://127.0.0.1:44455/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    }
  ],
  "paras": {
    "100": [
      {
        "name": "collator01",
        "wsUri": "ws://127.0.0.1:42149",
        "prometheusUri": "",
        "parachainId": 100
      }
    ]
  },
  "nodesByName": {
    "alice": {
      "name": "alice",
      "wsUri": "ws://127.0.0.1:36755",
      "prometheusUri": "http://127.0.0.1:35527/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    "bob": {
      "name": "bob",
      "wsUri": "ws://127.0.0.1:40181",
      "prometheusUri": "http://127.0.0.1:45287/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    "charlie": {
      "name": "charlie",
      "wsUri": "ws://127.0.0.1:33939",
      "prometheusUri": "http://127.0.0.1:45187/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    "dave": {
      "name": "dave",
      "wsUri": "ws://127.0.0.1:33299",
      "prometheusUri": "http://127.0.0.1:44455/metrics",
      "userDefinedTypes": {
        "Header": {
          "number": "u64",
          "parent_hash": "Hash",
          "post_state": "Hash"
        }
      }
    },
    "collator01": {
      "name": "collator01",
      "wsUri": "ws://127.0.0.1:42149",
      "prometheusUri": "",
      "parachainId": 100
    }
  }
}
```
The options we have at the moment is either to manually try to find the parachain spec by tailoring the chainspec or ask from @pepoviola to add that to the next version (or both).

I will open an issue at `zombienet` repo to address that missing spec.
Shall we wait for the issue to be resolved, or try to manually tailor the path of the parachain spec?

Reply:
Does `substrate-connect` support using the `adder-collator` or we need to use the `polkadot-collator` instead? Also, we can replace `wococo-local` to other chain. Two validators / one parachain is ok to this test?
Thanks!

Reply:
Im not sure about the first part of your question. But for the parachain test it would be awesome to have westend or rococo as relay chain. In case of Rococo I assume Wococo will do as a parachain. For westend and Rococo we already have the specs needed.

Reply:
The `adder-collator` should work but with limited capabilities (some JSON-RPC functions will return errors).